### ISSUE TITLE
algortihmic.sty not found

### ISSUE THREAD

**Issue by pirlite2** [texstudio-org/texstudio#1581]
### Environment
- TeXstudio: 3.1.1
- Qt: 5.12.8
- OS: Linux Mint 20
- TeX distribution: texlive 2020

### Description: I have a minimal (wizard-generated) Tex file of:

\documentclass[10pt,a4paper]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[english]{babel}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage[left=2.00cm, right=2.00cm, top=2.00cm, bottom=2.50cm]{geometry}

\usepackage{algorithmic}

\begin{document}

A minimum working example!
	
\end{document}

TXS was installed out-of-the-box. When I try to build this MWE within TXS, I am getting the  error of "File `algorithmic.sty' not found". OK. But this sty file is most definitely installed. When I build the file from the command line with "pdflatex MWE.tex", it works fine, which implies to me that the problem is not with my TexLive distribution. That seems to be set up correctly. 
No problems with other .sty files (that I am aware of)

If I comment out the \usepackage{algorithmic} line, it works. Many of the sty files are on the same path of /usr/local/texlive/2020/texmf-dist/tex/latex/ where things like graphicx.sty live. It can, for example, find graphicx.sty OK. So things seem to be going wrong at the final, package-name sub-directory.  I know algorithmic.sty is an unusual  case that lives, with algorithm.sty, in a sub-directory called algorithms (note: the plural). 

Any ideas? I cannot see any search paths, etc  in TXS that I am setting incorrectly or have failed to set. In fact, I have  similar software running on Linux Mint 18.3, identical TXS version 3.1.1 (built against Qt 5.5.1), and Texlive 2017 running on another machine, and the MWE above builds fine from within TXS on that computer!

Peter

**Comment by dbitouze** [comment]
Not a TXS question. See e.g.[this question/answers]( https://tex.stackexchange.com/q/1137/18401).

**Comment by pirlite2** [comment]
Sorry, but I think it is a TXS question.

'algorithms' is not some left-field style file - it comes with the base install of TexLive. (The Stackoverflow link you cited relates to non-standard style and class files.)  I maintain it is a TXS issue because I am unable to access a style file that is part of the base TexLive install (although I am able to access other such files.)

Peter

**Comment by dbitouze** [comment]
> 'algorithms' is not some left-field style file - it comes with the base install of TexLive.

But you mentioned `\usepackage{algorithmic}`, not `\usepackage{algorithmis}`.

**Comment by pirlite2** [comment]
Please see my original post: 

"algorithmic.sty is an unusual case that lives, with algorithm.sty, in a sub-directory called algorithms (note: the plural)."

So there is a directory called /usr/local/texlive/2020/texmf-dist/tex/latex/algorithms/ which contains two files: algorithmic.sty  and algorithm.sty.

FWIW: graphicx.sty also lives on /usr/local/texlive/2020/texmf-dist/tex/latex/ but in a sub-directory called graphics. Except TXS can find this fine despite the deviation from the normal Latex practice of placing the style file in a sub-directory of the same name.

So: why can't TXS find either algorithmic.sty or algorithm.sty whereas running pdflatex from the command line can?

**Comment by dbitouze** [comment]
> So: why can't TXS find either algorithmic.sty or algorithm.sty whereas running pdflatex from the command line can?

In a terminal, what returns:
```bash
kpsewhich algorithmic.sty algorithm.sty
```
and does the compilation of the `.tex` (say `test.tex`) file run smoothly:
```bash
pdflatex test
```

**Comment by pirlite2** [comment]
kpsewhich algorithmic.sty algorithm.styreturns:

/usr/local/texlive/2020/texmf-dist/tex/latex/algorithms/algorithmic.sty
/usr/local/texlive/2020/texmf-dist/tex/latex/algorithms/algorithm.sty

which backups my point that my TexLive install seems OK.

As I stated in my original post, running "pdflatex MWE.tex" in the shell completes without error and produce the expected pdf. I can see pdflatex reporting that it has found algorithmic.sty on the expected path. So I am pretty convinced my Tex installation is working fine.

**Comment by dbitouze** [comment]
> As I stated in my original post, running "pdflatex MWE.tex" in the shell completes without error and produce the expected pdf. I can see pdflatex reporting that it has found algorithmic.sty on the expected path. So I am pretty convinced my Tex installation is working fine.

OK, so indeed a TXS problem, sorry. I don't see what could be wrong with it, so I let the developers answer.

**Comment by sunderme** [comment]
check help/check latex installation

start txs from commandline and check if the issue persists

my guess is that you have *2* texlive installations and the GUI path (path when starting programs from gui) variable does not prefer usr/local

**Comment by pirlite2** [comment]
Ah! Great call. I somehow had parts of the texlive 2019 package from the Ubuntu repo installed. Adamant that I did not install that since the Ubuntu repo version is usually out of date, and because this is a clean install, I manually installed TexLive 2020. I can only think the incomplete texlive 2019 stuff was installed to satisfy  some package dependency and I didn't notice it flashing by.
So solved!
Many thanks,
Peter
